ProjectSidewalk/SidewalkWebpage
# Make gallery expanded view description scrollable

Resolves #3622

Previously, when the height was small enough, the description would overflow the expanded view modal. This is now fixed by setting the overflow property in the description body to `auto` and also some minor JS scripting to deal with the cases where css alone is not enough (in some situations where the height is very very small).

##### Before/After screenshots (if applicable)

![image](https://github.com/user-attachments/assets/4fb3703f-fb29-4832-832a-b354040b6c99)
![image](https://github.com/user-attachments/assets/e92c0ff3-271e-4f38-9f44-aa597886bbf2)

##### Testing instructions
1. Go to the gallery
2. Find an entry with an extremely long description. (Make one if you can't find something)
3. Confirm that it's now scrollable
4. You can also try resizing the window to see how it adapts.

##### Things to check before submitting the PR
